python/observation-publisher/tests/test_observability_backend.py

```diff
@@ -1,70 +1,155 @@
 from datetime import datetime
+from typing import List
 
 import pandas as pd
 import pyarrow as pa
+from arize.pandas.validation.errors import ValidationError
 from arize.pandas.validation.validator import Validator
-from arize.utils.types import Environments
-from arize.utils.types import ModelTypes as ArizeModelType
-from merlin.observability.inference import (BinaryClassificationOutput,
-                                            InferenceSchema, InferenceType,
-                                            ValueType)
+from arize.utils.types import (
+    ModelTypes as ArizeModelType,
+    Environments as ArizeEnvironment,
+    Schema as ArizeSchema,
+)
+from merlin.observability.inference import (
+    BinaryClassificationOutput,
+    InferenceSchema,
+    ValueType,
+    RankingOutput,
+    ObservationType,
+)
+from publisher.config import ArizeConfig
+from publisher.observability_backend import ArizeSink
 
-from publisher.observability_backend import map_to_arize_schema
 
+def assert_no_validation_errors(errors: List[ValidationError]):
+    try:
+        assert len(errors) == 0
+    except AssertionError:
+        for error in errors:
+            print(error.error_message())
+        raise
 
-def test_arize_schema_mapping():
+
+def assert_arize_schema_validity(
```

Comment:
should we use [no op arize client](https://github.com/Arize-ai/client_python/blob/main/tests/pandas/test_pandas_logger.py#L55C1-L55C1) to check whole log functionality?

Reply:
Yup, that's a good idea.